packages/esm-ward-app/src/ward-view/ward-view.resource.ts

```diff
@@ -22,3 +22,18 @@ export function filterBeds(admissionLocation: AdmissionLocation): BedLayout[] {
     .sort((bedA, bedB) => collator.compare(bedA.bedNumber, bedB.bedNumber));
   return bedLayouts;
 }
+
```

Comment:
This should ultimately be named `getWardMetrics` and it will need different input arguments and logic in the future.  For now, just rename the method, and add a prominent TODO comment indicating that this implementation will need to change once appropriate backend API endpoints are available.

Reply:
Made the change.Thanks @mseaton .